src/ol/AssertionError.js

```diff
@@ -13,13 +72,7 @@ class AssertionError extends Error {
    * @param {number} code Error code.
    */
   constructor(code) {
-    const path = VERSION === 'latest' ? VERSION : 'v' + VERSION.split('-')[0];
-    const message =
-      'Assertion failed. See https://openlayers.org/en/' +
-      path +
-      '/doc/errors/#' +
-      code +
-      ' for details.';
+    const message = `Assertion failed. ${messages[code]}.`;
```

Comment:
I'd be in favor of simplifying this to just `messages[code]`.  I think the "Assertion failed" part of the message isn't meaningful and potentially confusing to users.